cpp/src/aztec/honk/sumcheck/sumcheck_round.test.cpp

```diff
@@ -22,11 +22,11 @@ namespace test_sumcheck_round {
 TEST(SumcheckRound, ComputeUnivariateProver)
 {
     const size_t num_polys(proving_system::StandardArithmetization::NUM_POLYNOMIALS);
-    const size_t multivariate_d(1);
+    // const size_t multivariate_d(1);
```

Comment:
Looks like some of the commented out stuff in this file can just be deleted? can be done in follow-on

Reply:
Yeah, some of it can go, but it might be beneficial to preserve some of the info in comments (maybe it adds context to make the example more legible).